packages/teams-js/src/public/interfaces.ts

```diff
@@ -885,10 +905,6 @@ export interface DialogInfo {
  */
 export type TaskInfo = DialogInfo;
 
-export interface DialogSize {
```

Comment:
>DialogSize [](http://example.com/codeflow?start=17&length=10)

How was this in here twice? Shouldn't that have triggered some kind of compiler error?

Reply:
TypeScript will not throw an error as it is valid to have interfaces with the same name.

Reply:
🤯